content/js2/blocks/assembly/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = '🧱 Assembling the parts'
+headless = true
+time = 10
+facilitation = false
+emoji= '🧩'
+[objectives]
+1='Identify a bug when a function gives an unexpected output'
++++
+
+Now suppose we have a program where we use the functions we implemented earlier:
+
+```js
+const salaries = [10, 20, 30, 40, 60, 80, 80];
+const median = calculateMedian(salaries);
+const mean = calculateMean(salaries);
+
+console.log(`The median salary is ${median}`);
+console.log(`The mean salary is ${mean}`);
+```
+
+{{<note type="activity" title="Run it">}}
+Try running the code above on your local machine. Check the console output. What do you see?
```

Comment:
I think we should either have the trainees _predict and write down_ an expectation, or we should tell them what our expectation is. Otherwise just telling them "What do you see?" doesn't necessarily prompt that something is wrong. Let's keep explicitly training "predict + observe" as a behaviour.